src/frontend/displaywindow/cdisplaywindow.cpp

```diff
@@ -279,6 +279,9 @@ void CDisplayWindow::insertKeyboardActions( BtActionCollection* a ) {
     actn = new QAction( /* QIcon(CResMgr::displaywindows::general::findStrongs::icon), */ tr("Strong's Search"), a);
     actn->setShortcut(CResMgr::displaywindows::general::findStrongs::accel);
     a->addAction(CResMgr::displaywindows::general::findStrongs::actionName, actn);
+
+    actn = new QAction(QIcon(), tr("Speak selected text"), a);
```

Comment:
The `QIcon()` argument is not needed here.

Reply:
Indeed. I've dropped it.